app/helpers/hosts_and_hostgroups_helper.rb

```diff
@@ -50,4 +50,10 @@ def multiple_filter(hosts)
                      "Reminder: <strong> All %{count} hosts are selected </strong> for query filter %{query}", host_count).html_safe % {count: host_count, query: h(params[:search]) }
     params[:search].blank? ? no_filter : with_filter
   end
+  
+  def proxy_openscap_enabled?(proxy_id)
+    return true if proxy_id.nil?
```

Comment:
I'd say that if no ID is provided, is doesn't have openscap.

Reply:
Correct. I was thinking to handle this in host and hostgroups _form, like only go to helper if there is openscap proxy selected, but I was unable to put that in nested if statement. Like,

if openscap_proxy?
 proxy_openscap_enabled?(proxy_id) 

So I put that in  proxy_openscap_enabled? and then negate condition on _form . If you think this could handle in another way then suggestions are welcome.